resources/carrier/ru/374.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2013 The Libphonenumber Authors
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Manually translated
+# http://en.wikipedia.org/wiki/Telephone_numbers_in_Armenia#Mobile_network_codes
+
+37441|Оранж
+37443|Билайн
+37449|ВиваСелл-МТС
+37455|Оранж
+37477|ВиваСелл-МТС
+37491|Билайн
+37493|ВиваСелл-МТС
+37494|ВиваСелл-МТС
+37495|Оранж
```

Comment:
Unblocking this review - @mhamlet shared evidence that Orange ranges now belong to Ucom: http://www.orange.com/en/Press-and-medias/press-releases-2016/press-releases-2015/Orange-signs-an-agreement-with-Ucom-for-the-sale-of-100-of-its-shares-in-Orange-Armenia (thanks @mhamlet!)

@armleo please update the Orange ranges to Ucom.